consensus/snapshot/snapshot.c

```diff
@@ -76,9 +81,123 @@ static retcode_t iota_snapshot_initial_state(snapshot_t *const snapshot,
  * Public functions
  */
 
+retcode_t iota_snapshot_init_conf(char const *const snapshot_conf_file,
+                                  snapshot_conf_t *const conf, bool testnet) {
+  retcode_t ret = RC_OK;
+  FILE *file = NULL;
+  size_t len = 0;
+  char *content = NULL;
+  cJSON *json = NULL, *tmp = NULL, *timestamp = NULL, *signature = NULL,
+        *coordinator = NULL;
+
+  if ((file = fopen(snapshot_conf_file, "r")) == NULL) {
+    log_error(SNAPSHOT_LOGGER_ID, "Snapshot configuration file not found\n");
+    ret = RC_SNAPSHOT_FILE_NOT_FOUND;
+    goto done;
+  }
+
+  if (fseek(file, 0, SEEK_END) < 0 || (len = ftell(file)) < 0 ||
```

Comment:
shouldn't `ftell` be compared to `<=` 0?

Reply:
Indeed